docs/guides/sqlite3_db_with_concord.md

```diff
@@ -0,0 +1,265 @@
+    log_fatal("[SQLITE] Something went wrong while creating concord_guides table. [%s]", msgErr);
+    if (sqlite3_close(db) != SQLITE_OK) {
+        log_fatal("[SQLITE] Failed to close sqlite db. [%s]", sqlite3_errmsg(db));
+        abort();
+    }
+    return;
+}
+
+sqlite3_close(db);
+```
+
+Here, we are defining the command that **will be executed** with sqlite3_mprintf, the command going to be executed is "`CREATE TABLE IF NOT EXISTS concord_guides(user_id INT, guild_id INT)`", where **if the concord_guides table doesn't exist**, it is going to be created.
+
+Also, we are saving **INT values**, but if you wanted to **save a text value (like a string)**, you can simply replace `INT` with `TEXT` (WARNING: **This will cause you to have to do some modifications when retrieving data**).
+
+Then `sqlite3_exec` is going to perform the command defined with `sqlite3_mprintf`, and, **if some error happens**, it is going to write on the `char *msgErr`. After `sqlite3_exec`, we are going to free the query (generated by `sqlite3_mprintf`) to **not cause a memory leak**.
+
+After freeing the query, we **check if the status code of the `sqlite3_exec` is not equal to `SQLITE_OK`**, in case it's not, it's going to log a fatal saying that it failed that something failed while creating the table, and also includes the error message and closing the DB.
+
+Done! If all goes well, then you've created a table called concord_guides.
+
+## Saving records into the created table
+
+Now that we created a table, we need to save records into it, it's the purpose of sqlite3.
+
+For this, it will be pretty similar to the way that we created the table. Follow the example:
+
+```c
+query = sqlite3_mprintf("INSERT INTO concord_guides(user_id, guild_id) values(%"PRIu64", %"PRIu64");", msg->author->id, msg->guild_id);
+rc = sqlite3_exec(db, query, NULL, NULL, &msgErr);
+
+sqlite3_free(query);
+
+if (rc != SQLITE_OK) {
+  log_fatal("[SQLITE] Something went wrong while inserting values into concord_guides table. [%s]", msgErr);
+  if (sqlite3_close(db) != SQLITE_OK) {
+        log_fatal("[SQLITE] Failed to close sqlite db. [%s]", sqlite3_errmsg(db));
+        abort();
+    }
+}
+
+sqlite3_close(db);
+```
+
+Pretty similar, isn't it? The difference is, the command now is "`INSERT INTO concord_guides(user_id, guild_id) values(%"PRIu64", %"PRIu64");`".
+
+The `user_id, guild_id` inside the () are the same that we put while creating the table, **it MUST match the one put when we created it**, after, in the () of the values we defined the value as the ID of the author of the command and the ID of the guild that the command has been executed.
+
+After that, we did the same thing we did when creating the table, freed the generated query, verified if the status code of the `sqlite3_exec` is not equal to `SQLITE_OK`, and if it's not, log a fatal error and closes the DB. if it fails to close the DB, just abort. (NOTE: If you want a better explanation, see `Creating a sqlite3 table` above)
```

Comment:
```suggestion
```
Since this information has already been talked about previously, it can just be left out here!